tests/bugs/spirv-opt-SROA-of-globals.slang

```diff
@@ -0,0 +1,32 @@
+//TEST:SIMPLE(filecheck=CHECK): -target spirv -fvk-use-entrypoint-name
+// Note: SROA will only run for 100 elements if spirv-opt does not manually set the spirv-opt `scalar-replacement` option
+//CHECK-NOT: OpCompositeConstruct %Data
```

Comment:
I think this test needs a better description of what it is testing.
I am talking about the summary version of description not the details.